mitosheet/src/hooks/useSendEditOnClick.tsx

```diff
@@ -63,6 +67,7 @@ function useSendEditOnClick<ParamType, ResultType>(
         (args: any) => {
             _setParams(args); // update the params
             setParamsApplied(false); // mark them as not applied
+            setAttemptedEditWithTheseParamsMultipleTimes(false); // mark the user as not having to apply the edit multiple times
```

Comment:
"// mark the user as not having to apply the edit multiple times" -> "mark the user as not having applied the same edit multiple times"